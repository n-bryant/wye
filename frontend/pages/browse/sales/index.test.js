import { BrowseSalesPage } from "./index";

describe("BrowseSalesPage", () => {
  const props = {
    calsses: {
      root: "root"
    }
  };

  it("should render successfully", () => {
    const wrapper = shallow(<BrowseSalesPage {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
