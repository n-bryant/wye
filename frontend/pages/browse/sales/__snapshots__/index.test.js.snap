// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrowseSalesPage should render successfully 1`] = `
<div
  className="wye-BrowseSalesPage"
>
  <WithWidth(WithStyles(GamesFilterWidget))
    initialFilters={
      Object {
        "filters": Object {
          "gameFilters": Object {
            "onSale": true,
          },
        },
        "orderBy": Array [
          "DISCOUNT",
          "USER_RATING",
        ],
        "sortOrder": "DESC",
      }
    }
    subtitle="Browse titles available at a discount"
    title="Sales"
  />
</div>
`;
